client/src/components/Lesson/LessonCreator.tsx

```diff
@@ -60,14 +58,6 @@ export class LessonCreator extends BindingComponent<LessonCreatorProps, Lesson>
         return this.state;
     }
 
-    componentWillMount() {
-        if (this.props.notFound) {
```

Comment:
We should probably remove this prop as well if we aren't using it anymore (which I think is the case)